### PR TITLE
#50 added docker-compose and Docker for Postgresql and Redis; updated…

### LINKED ISSUE #50: Implement Docker for PostgreSQL and Redis
To make development life more manageable, implement Docker for the PostgresSQL database and Redis.

### PR DESCRIPTION
**Pull request by vygandas**
Fix #50

### CODE REVIEW

**Review by CristoferPortela — APPROVED** [review]

**Comment by jerrythimothyj** [comment]
Should we follow a template for the Pull request? That way we can understand what the context is, a gif or 1 min video that explains what has been changed and how it helps bla bla...?

File: .dockerignore [hunk 1]
```diff
@@ -8,3 +8,7 @@ docker-compose.yml
 Dockerfile
 endpoints.json
 README.md
+node_modules
+tmp
```
Comment (jerrythimothyj) [inline .dockerignore:12]: ? What does this do? How is this created?

File: README.md [hunk 2]
```diff
@@ -139,8 +139,10 @@ Install Isomera with this **Quickstart** command to create a Isomera project ins
 - (Use **yarn** to install the Isomera project (recommended). [Install yarn with these docs](https://yarnpkg.com/lang/en/docs/install/).)
 
 ```bash
+docker-compose up -d
 yarn install
-yarn start
+cp .env.example .env
```
Comment (jerrythimothyj) [inline README.md:144]: Probably automate these steps into 1 command? a shell command may be?

File: .dockerignore (hunk 1 above)
Comment (vygandas) [reply .dockerignore:12]: Oh, I just added because seemed like it would make sense

File: README.md (hunk 2 above)
Comment (vygandas) [reply README.md:144]: Well, it's 1 time ever run, thinking might be enough to do manually that line 🤔

**Pull request merged**